Avoid installing META files for camlp4, labltk, and num.
Those are provided by their respective packages now.

Index: findlib-1.8.0/configure
===================================================================
--- configure.orig
+++ configure
@@ -576,7 +576,7 @@
 
 # Generate the META files now.
 
-l="$ldbm dynlink $lgraphics $lnum str threads unix stdlib bigarray ocamldoc $llabltk $lcamlp4 $lobuild $lcomplibs $lbytes $lspacetime"
+l="$ldbm dynlink $lgraphics str threads unix stdlib bigarray ocamldoc $lobuild $lcomplibs $lbytes $lspacetime"
 
 for dir in site-lib-src/*; do
     # We do not really know if $dir is a directory.
